examples/starter/crds/config.yaml

```diff
@@ -410,6 +418,22 @@ spec:
         # use 'both' for sending slack notification whether component upgrade is success or failure
         criteria: failure
 
+        # [optional] override default extra message to when sending with componentUpgrade report
+        # if you want to tag users, groups in slack. you have to sending with the slack format
+        # use `<@UserId>` for mention users in the message e.g. "Hello <@U024BE7LH>, please check the report"
+        # use `<!subteam^ID>` for mention groups in the message e.g. "<!subteam^SAZ94GDB8>, please help check."
+        # additional format: https://api.slack.com/reference/surfaces/formatting#linking_to_channels_and_users
+        extraMessage: "ComponentUpgrade Extra Message"
+
+      activePromotion:
+        # [optional] override default extra message to when sending with activePromotion report
+        # if you want to tag users, groups in slack. you have to sending with the slack format
+        # use `<@UserId>` for mention users in the message e.g. "Hello <@U024BE7LH>, please check the report"
+        # use `<!subteam^ID>` for mention groups in the message e.g. "<!subteam^SAZ94GDB8>, please help check."
+        # additional format: https://api.slack.com/reference/surfaces/formatting#linking_to_channels_and_users
+        extraMessage: "ComponentUpgrade Extra Message"
```

Comment:
This one as well. 
Also, it should be Active Promotion if it will be referred to